templates/answerfile_4.2_upgrade.txt.j2

```diff
@@ -0,0 +1,15 @@
+{% include "./templates/basic_answerfile.txt.j2" %},
```

Comment:
missing answers which exist in 4.1 upgrade answer file.
you need to include the 4.1_upgrade answer file or create a basic answer file for both

Reply:
Fixed